lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -182,6 +221,36 @@ function getById(id, callback) {
     });
 }
 
+/**
+* List all the configGroups created in the IoT Agent for a service and a subservice.
+*
+* @param  {String} service        Service used to filter the groups.
+* @param  {String} subservice     Subservice used to filter the groups.
+* @param  {Function} callback     The callback function.
+*/
+function findConfigGroups(service, subservice, callback) {
+   const condition = {};
+
+    function toObjectFn(obj) {
+        return obj.toObject();
+    }
+
+    condition.service = service;
+    condition.subservice = subservice;
+
+    const query = Group.model.find(condition).sort();
+
+    async.series([query.exec.bind(query), Group.model.countDocuments.bind(Group.model, condition)], function (
+        error,
+        results
+    ) {
+        callback(error, {
+            count: results[1],
+            configGroups: results[0].map(toObjectFn)
+        });
+    });
+}
+
```

Comment:
Duplicated code, same as `function find(service, subservice, callback) `. No required

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11